TombEngine/Game/effects/effects.cpp

```diff
@@ -1588,9 +1633,9 @@ void ProcessEffects(ItemInfo* item)
 
 		auto pos = GetJointPosition(item, 0);
 		TriggerDynamicLight(pos.x, pos.y, pos.z, falloff,
-			std::clamp(Random::GenerateInt(-32, 32) + int(item->Effect.LightColor.x * UCHAR_MAX), 0, UCHAR_MAX),
```

Comment:
I would not directly use 0-255 range for colors in `Vector3` type. We are trying to gradually get rid of old 0-255 RGB range conversion and migrate to normalized 0-1 range. This will allow easy blending operations in future (multiplying, bounds checking, directly passing color values to shader which also uses normalized range).

Reply:
changed to 0.0f - 1.0f ranged float